Source: Upstream
Upstream: Yes, r968314
Reason: Kopete warns every time you connect to msn when you've added yourself
        to the contacts list.

From b0ff2f2cd185c68997e955960e902a7597cfe968 Mon Sep 17 00:00:00 2001
From: rjarosz <rjarosz@283d02a7-25f6-0310-bc7c-ecb5cbfe19da>
Date: Fri, 15 May 2009 13:49:22 +0000
Subject: [PATCH] Allow myself as msn allows this.

git-svn-id: svn://anonsvn.kde.org/home/kde/trunk/KDE/kdenetwork@968314 283d02a7-25f6-0310-bc7c-ecb5cbfe19da
---
 kopete/protocols/wlm/wlmprotocol.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/kopete/protocols/wlm/wlmprotocol.cpp b/kopete/protocols/wlm/wlmprotocol.cpp
index abff329..bcb455e 100644
--- a/kopete/protocols/wlm/wlmprotocol.cpp
+++ b/kopete/protocols/wlm/wlmprotocol.cpp
@@ -38,7 +38,7 @@ K_PLUGIN_FACTORY (WlmProtocolFactory, registerPlugin < WlmProtocol > ();
     WlmProtocol::WlmProtocol (QObject * parent,
                               const QVariantList & /*args */ )
     :
-Kopete::Protocol (WlmProtocolFactory::componentData (), parent),
+Kopete::Protocol(WlmProtocolFactory::componentData (), parent, true),
 wlmOnline (Kopete::OnlineStatus::Online, 25, this, 1, QStringList (),
            i18n ("Online"), i18n ("O&nline"),
            Kopete::OnlineStatusManager::Online),
-- 
1.6.3.1

